### DISCUSSION TITLE
Cannot type Expiry date

### DISCUSSION THREAD

**Discussion by stdedos** [keepassxreboot/keepassxc#10605] [Q&A] [answered]
Am I doing something wrong here?

https://github.com/keepassxreboot/keepassxc/assets/133706/355f6a25-c53e-4883-9dbf-d8635dcab6d5

I thought it should've worked?
I tried pressing Enter - but that saves the entry.

Using the arrow keys works, but it might be a bit slow :sweat_smile:

**Comment by droidmonkey** [comment, marked as answer]
works for me with the standard US layout for date/time. This is probably a Qt bug since this is a standard Qt Date/Time Widget. You can also press the arrow button on the far right to drop down a date picker.
